<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Deiby Estrada</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>592</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>623</width>
    <height>592</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>QUIZZZZ</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>python1.jpg</normaloff>python1.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>10.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 48, 48);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnIniciar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>150</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
gridline-color: rgb(255, 85, 0);
background-color: rgb(147, 147, 147);</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Nombre">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 171, 171);</string>
    </property>
    <property name="locale">
     <locale language="Spanish" country="Spain"/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSiguiente">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(129, 129, 129);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>SIGUIENTE</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblOpciones">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>230</red>
          <green>248</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>230</red>
          <green>248</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>230</red>
          <green>248</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>230</red>
          <green>248</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>230</red>
          <green>248</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>230</red>
          <green>248</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>230</red>
          <green>248</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>230</red>
          <green>248</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>230</red>
          <green>248</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 248, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>5</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::CustomDashLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>4</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>350</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
   </widget>
   <widget class="QLineEdit" name="Puntaje">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(48, 48, 48);
color: rgb(255, 0, 0);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblPuntaje">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>PUNTAJE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>ACEPTA</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="brrProgreso">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:1px;
alternate-background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombre">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="numeroPreguntas">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(177, 177, 177);</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pregunta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(189, 189, 189);
border-radius:10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
